part_1/title_IX/sekcja_56_PL.md

```diff
@@ -0,0 +1,35 @@
+
+<a name="section56"> </a><p align="center">Artykuł 56</p>
+
+<p align="center">Zatrzymanie broni palnej</p>
+
+(1) Funkcjonariusz policji ma prawo zatrzymać: broń kategorii A, B, C lub D, amunicję, myśliwski czarny proch, proch bezdymny, zapałki, pozwolenie, licencję na broń, świadectwo broni, listy przewozowe dotyczące stałego wywozu, przywozu lub tranzytu broni i amunicji, zezwolenie na wywóz produktów, które są objęte kontrolą handlu w Czechach ze względów bezpieczeństwa, Europejską Kartę Broni, pozwolenie na transport lub zgłoszenie transportu broni i amunicji z artykułu 50 lub 50a; aby zapobiec takim działaniom ich posiadaczy, które powodują nie spełnienie obowiązków lub łamamie zakazów ustanowionych ustawą.
+<!-- dosł: aby zapobiec ich właścicielom takich działań, które naruszają obowiązek lub nie przestrzegają zakazu ustanowionego w ustawie, ale to lepiej brzmi. Potrzebne znalezienie, czy dobrze przedłumaczyłam szczególnie 'černý lovecký prach, bezdýmný prach, zápalky', bo wydaje się to... dziwne. -->
+
+(2) Funkcjonariusz policji ma prawo zatrzymać: broń kategorii A, B lub C, amunicję do tej broni, pozwolenie, licencję na broń, świadectwo broni, listy przewozowe dotyczące stałego wywozu, przywozu lub tranzytu broni i amunicji, zezwolenie na wywóz produktów, które są objęte kontrolą handlu w Czechach ze względów bezpieczeństwa, Europejską Kartę Broni, w której są zapisane bronie z kategorii A, B lub C, pozwolenie na transport lub zgłoszenie transportu broni i amunicji z artykułu 50 lub 50a, jeśli
+
+a) istnieje uzasadnione podejrzenie, że u posiadacza pozwolenia na broń doszło do takiej zmiany stanu zdrowia, która może w związku z obsługą broni stanowić zagrożenie dla jego życia lub zdrowia albo życia lub zdrowia innych osób,
+
+b) cofnięto : pozwolenie na broń zgodnie z artykułem 27 ust. 1 punkt a), b) lub c); lub licencję na broń zgodnie z artykułem 36 ust. 1 punkt b) lub artykułem 36 ust. 2; a uprzedni posiadacz karty broni palnej lub pozwolenia na broń nie oddał świadectwa broni, broni, amunicji i zakazanych akcesoriów do broni właściwemu wydziałowi policji lub   
+<!-- potrzebne sprawdzenie w poprzednich artykułach: różnica między 'zbrojní průkaz' i  'průkaz zbraně', to pierwsze jest b) - ja przetłumaczyłam jako pozwolenie na broń -->
```

Comment:
To może być odpowiednik podobnych terminów w polskim prawie:
* `zbrojní průkaz` - pozwolenie na broń
* `průkaz zbraně` - świadectwo broni

Trzeba przetłumaczyć poprzednie przepisy, może terminy są jasno zdefiniowane i rozróżniane. W takim przypadku w tłumaczeniu też trzeba będzie konsekwentnie rozróżniać.